### PR TITLE
Adapt the physical model layer for Fuchsia

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@abarth

File: flow/layers/physical_model_layer.cc [hunk 1]
```diff
@@ -17,19 +22,32 @@ void PhysicalModelLayer::Preroll(PrerollContext* context, const SkMatrix& matrix
...
+  return bounds;
 }
 
+#if defined(OS_FUCHSIA)
+
+void PhysicalModelLayer::UpdateScene(SceneUpdateContext& context,
+                                     mozart::Node* container) {
+  context.AddLayerToCurrentPaintTask(this);
+  auto node = mozart::Node::New();
+  node->content_clip = mozart::RectF::From(rrect_bounds());
```
Comment (abarth) [inline flow/layers/physical_model_layer.cc]: We want to clip to `rrect_.getBounds()`, like we do in the non-mozart codepath.  The inflated bounds are to provide room for our shadow, but when clipping the children, we don't want to include the extra room.

Comment (jason-simmons) [inline flow/layers/physical_model_layer.cc]: done

**Comment by abarth** [comment]
LGTM

**Pull request merged**